docs/source/author/chapters.rst

```diff
@@ -68,9 +64,7 @@ In the following sections, each configuration file is exaplained in more detail.
 
 .. note::
 
-  - Some of the keys (:ref:`what-is-a-key`) have angle brackets around them,
-    ``<like so>``. This means that they are variables and you can call them
-    whatever you like in your configuration file (without the angle brackets).
+  - Some of the keys (:ref:`what-is-a-key`) have angle brackets around them, ``<like so>``. This means that they are variables and you can call them whatever you like in your configuration file (without the angle brackets).
```

Comment:
Mention that they should be consistent, i.e. replace every instance of <chapter-key> as exactly the same key